scripts/create-pr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+PR_TITLE="chore (automated): OpenSauced updates"
+BRANCH_NAME="chore_automated_open-sauced-updates_$(date +%s)"
+
+git branch $BRANCH_NAME
+git switch $BRANCH_NAME
+
+# There are potentially multiple files if the blog post has images.
+git add .
+
+# Check if there are any changes to commit
+if [[ -n "$(git status --porcelain)" ]]; then
+  echo "Creating PR \"$PR_TITLE\" for branch $BRANCH_NAME"
+  git commit -m "$PR_TITLE"
+  git push origin $BRANCH_NAME
+
+  # Attempt to create the PR (dry run)
+  if gh pr create --title "$PR_TITLE" --body "This is an automated PR generated by the OpenSauced pizza-action." --dry-run; then
+    # If dry run is successful, create the actual PR
+    PR_URL=$(gh pr create --title "$PR_TITLE" --body "This is an automated PR generated by the OpenSauced pizza-action.")
+    echo "PR created successfully: $PR_URL"
+
+    # Attempt to merge the PR
```

Comment:
Curious your thoughts on why we'd want to attempt to auto-merge this for people? My gut feeling is this would be pretty unexpected for people and 9/10 times they'd want to manually review the changes being made by `pizza` since it could have significant organization / workflow impacts.

Reply:
You had mentioned in #1 or on Slack that orgs probably have their own workflows and might use the pizza-action in a composite workflow to decide what they want to do with the pizza-cli output, so  I had I had made this optional (creating the PR, previously pushing to default branch).

Since this is an automated process, I thought if they have automerging enabled, we can just set the merge to auto. I can remove it though. We can see what feedback people provide. We'll be using the create PR option, but others might not.

Reply:
> You had mentioned in https://github.com/open-sauced/pizza-action/pull/1 or on Slack that orgs probably have their own workflows and might use the pizza-action in a composite workflow to decide what they want to do with the pizza-cli output

True. Admittedly, I'm unsure how people will _actually_ use this since there isn't really anything like it out there. Alot of projects / orgs have their own workflows as you mentioned and maybe they'll just want to "send it ™️ " each time. So, ^ mostly thinking out loud, I could be swayed one way or another! Dealers choice!!